README.md

```diff
@@ -9,3 +9,29 @@ For more information about the control container, including how to use it and ho
 
 You'll need Docker 17.06.2 or later, for multi-stage build support.
 Then run `make`!
+
+## Connecting to AWS Systems Manager (SSM)
+
+Starting from v0.5.0, users have the option to pass in their own activations for SSM. This is for users that want to set up on-premises virtual machines (VMs) in their hybrid environment as managed instances.
```

Comment:
Should this say "pass in their own activation information"?

Reply:
I'm not sure actually. The AWS documentation refers to them as Hybrid Activations, but both I think could be correct.